packages/hoppscotch-common/src/components/app/Banner.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div
+    class="relative flex items-center px-4 py-2 transition text-tiny group"
```

Comment:
The `group` class doesn't seem to serve a purpose here. Also, could you verify if `relative ` and `transition` classes are necessary?